pkg/services/ngalert/api/tooling/README.md

```diff
@@ -2,12 +2,16 @@
 
 [view api](http://localhost)
 
-This aims to define the unified alerting API as code. It generates OpenAPI definitions from go structs
+This aims to define the unified alerting API as code. It generates OpenAPI definitions from go structs, in the `tooling/definitions` directory.
```

Comment:
I realized mid-way through that there are a few things I'd like to improve about these docs/tooling:
- Document the entire second phase of the generation, which was missing.
- Add makefile targets for doing both generation phases in one step.
- Generate the FGAC routing as part of the generated API. It's weird to have to hardcode the route elsewhere for FGAC, and it'd be nice to describe the permissions in the swagger doc

So, I ended up keeping the changes here super minimal and am doing this in a separate branch.